<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <layout class="QHBoxLayout" name="formLayout">
     <item>
      <widget class="QLineEdit" name="labelField">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Название поля</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeEnum">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Текст</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Целое число</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рациональное число</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Время</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дата</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дата и время</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Флажок</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Переключатель</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="propertiesLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset resource="../../../src.qrc">
       <normaloff>:/src/src/delete.png</normaloff>:/src/src/delete.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../src.qrc"/>
 </resources>
 <connections/>
</ui>
